docs/celeborn_cli.md

```diff
@@ -0,0 +1,220 @@
+                           [--config-level=level] [--config-name=username]
+                           [--config-tenant=tenant_id] [--host-list=h1,h2,
+                           h3...] [--hostport=host:port] [--worker-ids=w1,w2,
+                           w3...] (--show-worker-info | --show-apps-on-worker |
+                           --show-shuffles-on-worker |
+                           --show-top-disk-used-apps |
+                           --show-partition-location-info |
+                           --show-unavailable-peers | --is-shutdown |
+                           --is-decommissioning | --is-registered |
+                           --exit=exit_type | --show-conf |
+                           --show-container-info | --show-dynamic-conf |
+                           --show-thread-dump)
+      --apps=appId           The application Id list seperated by comma.
+      --cluster=cluster_alias
+                             The alias of the cluster to use to query masters
+      --config-level=level   The config level of the dynamic configs
+      --config-name=username The username of the TENANT_USER level.
+      --config-tenant=tenant_id
+                             The tenant id of TENANT or TENANT_USER level.
+      --exit=exit_type       Exit the application with a specified type
+  -h, --help                 Show this help message and exit.
+      --host-list=h1,h2,h3...
+                             List of hosts to pass to the command
+      --hostport=host:port   The host and http port
+      --is-decommissioning   Check if the system is decommissioning
+      --is-registered        Check if the system is registered
+      --is-shutdown          Check if the system is shutdown
+      --show-apps-on-worker  Show applications running on the worker
+      --show-conf            Show worker conf
+      --show-container-info  Show container info
+      --show-dynamic-conf    Show dynamic worker conf
+      --show-partition-location-info
+                             Show partition location information
+      --show-shuffles-on-worker
+                             Show shuffles running on the worker
+      --show-thread-dump     Show worker thread dump
+      --show-top-disk-used-apps
+                             Show top disk used applications
+      --show-unavailable-peers
+                             Show unavailable peers
+      --show-worker-info     Show worker info
+  -V, --version              Print version information and exit.
+      --worker-ids=w1,w2,w3...
+                             List of workerIds to pass to the command. Each
+                               worker should be in the format host:rpcPort:
+                               pushPort:fetchPort:replicatePort.
+```
+
+## Ratis Shell
+
+Celeborn CLI supports ratis shell with `celeborn-ratis` command to operate the master ratis service. Details of ratis shell refer to [Celeborn Ratis Shell](celeborn_ratis_shell.md).
```

Comment:
we should add CLI commands for these new REST Apis as well. I think this would be more preferable to use than using celeborn-ratis shell script, since it can be done remotely without being on the host. Should we update this part after we add those commands to CLI?

Reply:
@akpatnam25, IMO, the functions of Ratis REST API are same as of which celeborn-ratis shell script. If there are different Rest API, I will update this section to support.

Reply:
sounds good.